src/components/view/Contact/index.js

```diff
@@ -19,7 +19,7 @@ class Contact extends Component {
                 <ContactNavigation
                     >
                 </ContactNavigation>
-                <div className='page'>
+                <div>
```

Comment:
This className was previously helpful for dealing with page sizes requiring scrolling. If you don't want it here because of use of flexbox, maybe we should be consistent and update this styling across all `.page`s?